app/src/main/java/org/kiwix/kiwixmobile/zim_manager/fileselect_view/effects/DeleteFiles.kt

```diff
@@ -40,18 +40,31 @@ data class DeleteFiles(private val booksOnDiskListItem: List<BookOnDisk>) :
 
   override fun invokeWith(activity: AppCompatActivity) {
     (activity as ZimManageActivity).cachedComponent.inject(this)
+
+    var name = "\n"
```

Comment:
I don't like this being a var, you could use `joinToString` or a `fold` where the accumulator is `StringBuilder.appendLn`, `reduce` could work as well but `joinToString` will definitely be easiest